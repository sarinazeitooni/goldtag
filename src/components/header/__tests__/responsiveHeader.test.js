import React from 'react';
import { shallow } from 'enzyme';
import ReponsiveHeader from "../ResponsiveHeader";
describe("ReponsiveHeader component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<ReponsiveHeader/>)
        expect(container).toMatchSnapshot();
    });
    it("it should render header-top-container-sticky",()=>{
        const container = shallow(<ReponsiveHeader/>);
        expect(container.find('div.header-top-container-sticky')).toHaveLength(1);
    });
});