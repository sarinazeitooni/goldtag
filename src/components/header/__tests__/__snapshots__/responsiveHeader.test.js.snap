// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReponsiveHeader component it should render a div 1`] = `ShallowWrapper {}`;
